### DISCUSSION TITLE
About coding convention

### DISCUSSION THREAD

**Discussion by trunglebka** [ithewei/libhv#177] [Q&A] [answered]
Hi, I'm in my journey to find handful networking framework in C++ (comming from netty)
IMO, this project is a good fit for me and seem like it is structured good. But after reading through some examples, I see that the coding convention is not consistent especially naming convention. Ex. file name, class name, function name...
Can you tell me the convention you are following so I can easy understand it? If not, I think it's a good point to define some convention to follow. :thinking:

**Comment by ithewei** [comment, marked as answer]
I'm glad you paid attention to this library. Because both c/c++ are used, the naming style does not seem to be uniform. C mainly uses the underscore method and `h or hv_` prefix, and c++ mainly uses the camel case method and `namespace hv`. If you think the reading is not smooth, looking forward your suggest or PR.

**Reply by trunglebka** [reply]
Oh, thanks for the answer, I'm just begin to learn C++ without knowledge of C. 
The question is silly now :D
